mailbox/api/src/test/java/org/apache/james/mailbox/events/EventDeadLettersContract.java

```diff
@@ -460,7 +460,17 @@ default void groupsWithFailedEventsShouldReturnEmptyWhenNoStored() {
         }
     }
 
-    interface EventDeadLettersHealthCheckContract extends EventDeadLettersContract {
+    interface HasEventContract extends EventDeadLettersContract {
+        @Test
+        default void hadEventShouldReturnFalseOnNothingStored() {
+
+        }
+
+        @Test
+        default void hadEventShouldReturnTrueOnStoredEvents() {
+
+        }
 
+//        TODO more test
```

Comment:
Todo spotted

Reply:
`containsEventsShouldReturnTrueEvenWithSomeRemoved()`
`containsEventsShouldReturnFalseWhenAllRemoved()`

Reply:
@nvduc91 and here too